api-admin/src/main/java/bio/terra/pearl/api/admin/service/participant/ProfileExtService.java

```diff
@@ -21,8 +23,13 @@ public ProfileExtService(AuthUtilService authUtilService, ProfileService profile
   /**
    * Updates the profile on behalf of the enrollee; requires a justification for auditing purposes.
    */
+  @EnforcePortalStudyEnvPermission(permission = "participant_data_edit")
```

Comment:
I'd love it if you tried creating an @EnforcePortalEnrolleePermission (possibly in a follow-on PR).  This pattern of authing an admin to a specific enrollee is very common. 

You'll create a EnforcePortalEnrolleePermission interface, and a PortalEnrolleeAuthContext.  the latter will extend PortalStudyEnvAuthContext since we'll want to auth the user to the enrolleee's study environment too. Then create a EnforcePortalStudyEnvPermissionAspect class to implement the auth check and set the enrollee on the authContext

Reply:
Seems non-trivial, so I'll do this in a follow up PR!